airflow/providers/amazon/aws/sensors/rds.py

```diff
@@ -61,7 +66,14 @@ def _check_item(self, item_type: str, item_name: str) -> bool:
         except ClientError:
             return False
         else:
-            return bool(items) and any(map(lambda s: items[0]['Status'].lower() == s, self.target_statuses))
+            status_field = self._check_status_field()
+            return bool(items) and any(
+                map(lambda status: items[0][status_field].lower() == status, self.target_statuses)
+            )
```

Comment:
Also, the `return` could be modified from `map` to the generator if you wish. 
```suggestion
            return bool(items) and any(items[0][status_field].lower() == status for status in self.target_statuses)
```

For me, both variants are well readable but maybe this one is better. WDYT?

Reply:
Thanks for the suggestions. I'd like to make these updates in a separate PR.
The formatting is the result of [black](https://black.readthedocs.io/en/stable/) (line too long)